src/app/store/system-config/system-config.effects.ts

```diff
@@ -29,8 +30,14 @@ export class SystemConfigEffects {
     }),
   ));
 
+  advancedConfigUpdated$ = createEffect(() => this.actions$.pipe(
+    ofType(advancedConfigUpdated),
+    map(() => this.window.location.reload()),
+  ), { dispatch: false });
```

Comment:
I don't think I like the idea of reloading the page on every advanced setting changed. There are a lot of settings that don't require this.
You can keep it in one place where it is needed, but a // TODO comment needs to be added as this is a hack that would need to be fixed later.

Reply:
actually it's only if few different places, one for advancedConfig updated and some fore different routes where we trigger /reboot or /failover.
I'll add comment to all the places, they are independent

Reply:
`advancedConfigUpdated` is dispatched whenever any of the advanced settings forms is edited. There is no need to reload the whole UI when I say change something in Console form.